shared/src/mobileMain/kotlin/org/jetbrains/kotlinconf/ui/BookmarksScreen.kt

```diff
@@ -40,11 +40,11 @@ import org.jetbrains.kotlinconf.ui.components.TabBar
 import org.jetbrains.kotlinconf.ui.components.VoteAndFeedback
 
 enum class Bookmark(override val title: String) : Tab {
-    PAST("past"), UPCOMING("upcoming")
+    PAST("Past"), UPCOMING("Upcoming")
```

Comment:
Don't you want to extract it to the strings resources?

Reply:
Could I have a composable constructor?

Reply:
in that case you are suppose to use string IDs and resolve it to text in a compose scene